mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/util/JournalParams.java

```diff
@@ -132,6 +132,22 @@ public JournalParams(JournalRecord.ActionType journalActionType,
           JournalRecord.ActionStatus.COMPLETED));
       }
     },
+    DI_INVENTORY_AUTHORITY_MATCHED {
```

Comment:
I don't think that we should log this event because it's not the end of the flow. It will be consumed by mod-inventory.
Instead of this event please add DI_INVENTORY_AUTHORITY_UPDATED event that is the end of the flow